.travis.yml

```diff
@@ -16,7 +16,7 @@ services:
   - docker
 
 addons:
-  firefox: "46.0"
+  firefox: "47.0"
```

Comment:
@aluu317 If we are switching to using `chrome` then we can probably remove this line altogether.